libra_toolbox/neutronics/vault.py

```diff
@@ -96,11 +88,11 @@ def build_vault_model(
         libraries=["ENDFB-8.0-NNDC"],
         set_OPENMC_CROSS_SECTIONS=True,
         particles=["neutron"],
-        destination=cross_sections_destination,
```

Comment:
Let's keep this cause I think we may need it for the automation in the BABY run repos

Reply:
@SteSeg can you revert this change then I can merge this